tests/robottelo/test_vm.py

```diff
@@ -6,9 +6,13 @@
 
 from robottelo import ssh
 from robottelo.config.base import DistroSettings
+from robottelo.constants import NO_REPOS_AVAILABLE
+from robottelo.constants import SM_OVERALL_STATUS
 from robottelo.vm import VirtualMachine
 from robottelo.vm import VirtualMachineError
 
+from unittest.mock import call, patch
```

Comment:
Looks like `reorder-python-imports` in `pre-commit` doesn't like that line - I guess it would prefer to have it split to two lines.

Best to install `pre-commit` locally and run it, then commit changes it did.